Yoast/wpseo-woocommerce
# Fix broken grunt configurations

## Summary

This PR can be summarized in the following changelog entry:

*  N/A

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Ensure you have the latest changes from the Wiki repository (`master` branch) pulled on your PC.
* Create a beta and ensure the version number of all the JS and plugin files are properly bumped.

Fixes #
